2001-3000/LOC2531DaqqiI.xml

```diff
@@ -48,11 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPlace>
             <place type="monastery">
```

Comment:
`<place type="monastery gadam">`

Reply:
But the others are also probably gadamat, should it be indicated? here the word gadam was introduced into one of the mane variants.